openapi/p4pa-auth.openapi.yaml

```diff
@@ -172,7 +172,9 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ClientDTO'
+                type: array
+                items:
+                  type: string
```

Comment:
it is not supposed to return just the clientId! It should return the entire DTO without the clientSecret (it will be used to populate the list of organization clients table, with at least clientId and clientName)

Reply:
see how to solve it into p4pa-doc/design/a2a/openapi.yml, use ClientNoSecretDTO schema, allign your openapi.yml